noxfile.py

```diff
@@ -36,12 +72,17 @@ def unit_tests(session: Session) -> None:
 
 @nox.session(name="integration-tests", python=False)
 def integration_tests(session: Session) -> None:
-    """Runs the all integration tests"""
-    _integration_tests(session)
+    """Runs the all integration tests
+
+    Args:
+        db_version: version to start, if none no db will be started.
+    """
```

Comment:
Running itests without db does not make sense. Better fail immediately if `db_version` is missing.